### DISCUSSION TITLE
最近发现reality不知算不算漏洞 间接可给主动探测

### DISCUSSION THREAD

**Discussion by kingwilliam** [XTLS/Xray-core#3885] [General]
简单苗述
server reality inbounds
```
"realitySettings": {
    "show": false,
    "dest": "x.x.x.x",
    "serverNames": ["www.abc.xyz"],
    .....
},
```

如用browser https://www.abc.xyz/login (path 什麽也没关系 /picture 也可) 因reality只会 forward 到 https://www.abc.xyz/ 所以path输入什麽也只会显示 "/" 主页

还是我设定不当 只想如有主动探测 可连带path到指定网站 还望指正

**Comment by RPRX** [comment]
？
